### ISSUE TITLE
questions about magento-configurable-simple

### ISSUE THREAD

**Issue by kevinm2** [organicinternet/magento-configurable-simple#235]
What is the latest version? Should the latest version be downloaded from Magento-Connect or here at Github? I copied the files from github to a test site and site crashed!

If the latest should be downloaded from github, what is the best way to install the plugin? 

Is it compatible 1.9.1.1?

**Comment by grosykevin** [comment]
Hello to you, it is fully compatible, I have install via magento connect

**Comment by kevinm2** [comment]
Hi grosykevin. Thank you for the reply.
I implemented it, following "https://github.com/organicinternet/magento-configurable-simple/issues/224".
Copied the "Configurable.php" to magento/app/code/community/OrganicInternet/SimpleConfigurableProducts/Catalog/Model/Resource/Eav/Mysql4/Product/Indexer/Price/
I re-indexed and didn't get the Green "re-indexed message" and I got the following.
![index_001](https://cloud.githubusercontent.com/assets/12448068/7788486/0fffd65e-01f1-11e5-9d81-ae4dc9ee6c70.JPG)
It has been showing the above message, "Processing" for past 15 minutes.  I can re-index all others but when I re-index "Product Prices" I get this error: "[an error occurred while processing this directive]".

How long does it take to process?

**Comment by grosykevin** [comment]
did you edit the configurable.php file as I said previously?

**Comment by kevinm2** [comment]
Yes I did.  Here is the updated "Configurable.php" (Left is the new one):

![index_002](https://cloud.githubusercontent.com/assets/12448068/7788547/e76f596e-01f3-11e5-8ba0-7aab118e4a88.JPG)
![index_003](https://cloud.githubusercontent.com/assets/12448068/7788548/ef855a72-01f3-11e5-8889-efe72fcf9842.JPG)

It's now about 35 minutes, and is still says processing.

**Comment by grosykevin** [comment]
Send me email : grosykevin@gmail.com
